src/connection.cpp

```diff
@@ -294,8 +411,6 @@ namespace oxen::quic
         }
     };
 
-    int64_t sent_counter = 0;
-
```

Comment:
Whoops, my bad here leaving that in in the udp refactor.

Reply:
ill re-add it